CONTRIBUTING.md

```diff
@@ -66,6 +66,34 @@ To reformat:
 yarn format
 ```
 
+To reformat incremental changes:
+
+```
+yarn prettier --write .prettierrc.json
+```
+
+## Guide generation
+
+The guide generation script is an Alpha state.
+
+Each json file in [guide-gen/guide-configs](guide-gen/guide-configs) releates to a specific concept that's expressed by a declartive configuiration reperesented by conglomerate of related markdown files. These configuration files generate the [Concepts](https://docs.temporal.io/docs/temporal/#) and [Application development](https://docs.temporal.io/docs/application-development-guide) guides by means of the [index.js](guide-gen/index.js) script.
```

Comment:
```suggestion
Each JSON configuration file in [guide-gen/guide-configs](guide-gen/guide-configs) relates to a specific concept and identifies related Markdown files. The [index.js](guide-gen/index.js) script uses these configuration files to generate guides such as [Concepts](https://docs.temporal.io/docs/temporal/#) and [Application development](https://docs.temporal.io/docs/application-development-guide).
```
I generalized the guides because we likely will have more.